chapter6_part4:/052_Mapping_Analysis/40_Analysis.asciidoc

sdlyjzh opened the pull request:
<!--
Thanks for the Pull Request!  We really appreciate your help and contribution!

Before you submit the PR, have you signed Contributor License Agreement: https://www.elastic.co/contributor-agreement/ ?

PR's (no matter how small) cannot be merged until the CLA has been signed.  
It only needs to be signed once, however. Thanks!
-->

Commit:
35_Inverted_index

Commit:
35_Inverted_index

Commit:
40_Analysis.asciidoc

Commit:
40_Analysis.asciidoc

Commit:
40_Analysis.asciidoc

medcl commented on 052_Mapping_Analysis/40_Analysis.asciidoc:
```diff
@@ -1,146 +1,105 @@
 [[analysis-intro]]
 [role="pagebreak-before"]
-=== Analysis and Analyzers
+=== 分析和分析器Analysis and Analyzers
```
和-》与 怎么样？
Analysis and Analyzers可以去掉

sdlyjzh replied:
已更改。

richardwei2008 committed:
chapter16_part8: /130_Partial_Matching/35_Search_as_you_type.asciidoc #Fix (#354)

* fix after merge

* improve

Commit:
chapter11_part1:/075_Inside_a_shard/10_Intro.asciidoc (#165)

* chapter11:/075_Inside_a_shard/10_Intro.asciidoc

* 补充缺失空格 && data store 仓库-->存储 && 调整语句表意

richardwei2008 committed:
chapter13_part2: /100_Full_Text_Search/05_Match_query.asciidoc #Fix (#355)

* fix after merge

* improve

高层 -> 高级

chenryn committed:
chapter45_part2:/500_Cluster_Admin/15_marvel.asciidoc (#50)

* chapter45_part2:/500_Cluster_Admin/15_marvel.asciidoc

* 删除『各』字

* 更正medcl提出的意见

* 修正biyuhao提供的意见

* 按照review意见修改dont翻译

richardwei2008 committed:
chapter16_part3: /130_Partial_Matching/10_Prefix_query.asciidoc (#103)

* chapter16_part3: /130_Partial_Matching/10_Prefix_query.asciidoc

初译

* improve

->“前缀越短所需访问的词越多。如果我们要以 W 作为前缀而不是 W1 ，那么就可能需要做千万次的匹配”

pengqiuyuan committed:
chapter22_part22:/300_Aggregations/125_Conclusion.asciidoc (#296)

* chapter22_part22:/300_Aggregations/125_Conclusion.asciidoc

初译

* chapter22_part22:/300_Aggregations/125_Conclusion.asciidoc

修改

* chapter22_part22:/300_Aggregations/125_Conclusion.asciidoc

修改

* chapter22_part22:/300_Aggregations/125_Conclusion.asciidoc

按照 review 建议修改。@medcl

* chapter22_part22:/300_Aggregations/125_Conclusion.asciidoc

第一行：[[_closing_thoughts]]

* chapter22_part22:/300_Aggregations/125_Conclusion.asciidoc

按 review 意见修改

* chapter22_part22:/300_Aggregations/125_Conclusion.asciidoc

移除多余前缀

* chapter22_part22:/300_Aggregations/125_Conclusion.asciidoc

按照 review 意见修改，感谢 @qindongliang

* chapter22_part22:/300_Aggregations/125_Conclusion.asciidoc

按照 review 意见修改，感谢 @smilesfc

medcl committed:
misc translation (#365)

richardwei2008 committed:
chapter12_part5: /080_Structured_Search/20_contains.asciidoc (#31)

* chapter12_part2: /080_Structured_Search/20_contains.asciidoc

chapter12_part2: /080_Structured_Search/20_contains.asciidoc
第二部分：第12章，结构化搜索，包含

* chapter12_part2: /080_Structured_Search/20_contains.asciidoc

fix: 手工换行问题

* 1. 词项 -> 项
2. 位集不译 -> bitset
3. “获取文档ID” 空格

* improve after merge

improve after merge

richardwei2008 committed:
chapter16_part4: /130_Partial_Matching/15_WildcardRegexp.asciidoc (#104)

* chapter16_part4: /130_Partial_Matching/15_WildcardRegexp.asciidoc

初译

* improve

file name

* improve

-> 通配/通配符

* improve

richardwei2008 committed:
chapter17_part12: /170_Relevance/60_Decay_functions.asciidoc (#123)

* chapter17_part12: /130_Partial_Matching/60_Decay_functions.asciidoc

初译

* improve

richardwei2008 committed:
chapter17_part15: /170_Relevance/75_Changing_similarities.asciidoc (#127)

* chapter17_part15: /170_Relevance/75_Changing_similarities.asciidoc

初译

* 修改

字段长规范化 》字段长度规范化（field-length normalization）

* improve

richardwei2008 committed:
chapter17_part16: /170_Relevance/80_Conclusion.asciidoc (#128)

* chapter17_part16: /170_Relevance/80_Conclusion.asciidoc

初译

* 修改

监控仪器 》 监控手段

* improve

luotitan committed:
chapter24_part3: /270_Fuzzy_matching/30_Fuzzy_query.asciidoc (#133)

* 第一次提交

* 第二次修改提交

* 修改数字两边的空格

* 按照nodesssreview意见修改

luotitan committed:
chapter24_part6: /270_Fuzzy_matching/60_Phonetic_matching.asciidoc (#153)

* 重新第一次提交

* 按照review意见调整

* 按照nodesssreview意见修改

Commit:
chapter26_part2:/310_Geopoints/30_Filter_by_geopoint.asciidoc (#182)

* 翻译

* 按review修改

tangmisi committed:
chapter9_part3: /060_Distributed_Search/10_Fetch_phase.asciidoc (#224)

* 第二次review修改

* fix id

* 去掉开头的空格

luotitan committed:
chapter9_part2: /060_Distributed_Search/05_Query_phase.asciidoc (#272)

* 查询阶段-第一次提交

* 根据review意见修改

* 根据Medclreview意见修改

lephix committed:
chapter2_part7:/020_Distributed_Cluster/30_Scale_more.asciidoc (#315)

* 第2章30小节

* 按照要求进行修改，恢复((()))隐藏内容

Commit:
chapter40_part2:/340_Geoshapes/72_Mapping_geo_shapes.asciidoc (#316)

Commit:
chapter40_part3:/340_Geoshapes/74_Indexing_geo_shapes.asciidoc (#317)

* chapter40_part3:/340_Geoshapes/74_Indexing_geo_shapes.asciidoc

* chapter40_part3:/340_Geoshapes/74_Indexing_geo_shapes.asciidoc

Commit:
chapter40_part4:/340_Geoshapes/76_Querying_geo_shapes.asciidoc (#322)

luotitan committed:
chapter41_part2: /400_Relationships/15_Application_joins.asciidoc (#351)

* 重新提交

* 根据tangmisi review意见修改

Commit:
chapter40_part5:/340_Geoshapes/78_Indexed_geo_shapes.asciidoc (#323)

medcl committed:
chapter/foreword: foreword.asciidoc (#364)

* first commit

* improve

* improve

* fix typo

* improve

* improve a bit

* improve

* improve

medcl committed:
Fix filename in _deleting_an_index.html (#367)

/070_Index_Mgmt/05_Create_Delete.asciidoc

luotitan committed:
chapter41_part6: /400_Relationships/26_Concurrency_solutions.asciidoc (#75)

* 第一次提交

* 第一次提交

* 根据review意见修改

* 重新整体检查修改后提交

* 根据medcl最新review意见修改

luotitan committed:
chapter24_part5: /270_Fuzzy_matching/50_Scoring_fuzziness.asciidoc (#154)

* 重新第一次提交

* 修改符号错误

* 按照nodesssreview意见修改

* 错字修改

richardwei2008 committed:
chapter14_part5: /110_Multi_Field_Search/25_Multi_match_query.asciidoc (#93)

* chapter14_part5: /110_Multi_Field_Search/25_Multi_match_query.asciidoc

初译

* 修改

脱字号 》 扬字符

* improve

* improve

默认情况下，下面查询的类型是 best_fields
->
默认情况下，查询的类型是 best_fields

luotitan committed:
chapter24_part1: /270_Fuzzy_matching/10_Intro.asciidoc (#106)

* 第一次提交

* 按照review意见调整

* 根据leo650review意见修改

* 根据nodesss review意见修改

dajyaretakuya committed:
chapter4_part2: /040_Distributed_CRUD/05_Routing.asciidoc (#137)

* Update 040_Distributed_CRUD/05_Routing.asciidoc

* Update 040_Distributed_CRUD/05_Routing.asciidoc

* Update 05_Routing.asciidoc

* Update 05_Routing.asciidoc

* Update 040_Distributed_CRUD/05_Routing.asciidoc

* 修正几处翻译错误 040_Distributed_CRUD/05_Routing.asciidoc

* 修正几处翻译错误 040_Distributed_CRUD/05_Routing.asciidoc

* 修正翻译错误：索引文档后->索引文档时

* 中英文空格

* “余数”这个词的样式由斜体换成黑体，因其样式容易被前面的_id干扰。（原因暂时不明，英文原文是没问题的）

* 路由一个文档到一个分片 and 数字加格式

* 采纳luotitan的review

yichao2015 committed:
chapter10_part5:/070_Index_Mgmt/25_Mappings.asciidoc (#298)

* chapter10_part5:/070_Index_Mgmt/25_Mappings.asciidoc

* 第一次review后修改

* 二次 review后修改

* 三次review后修改

* 重新修改

yichao2015 committed:
chapter10_part11:/070_Index_Mgmt/40_Custom_Dynamic_Mapping.asciidoc (#301)

* chapter10_part11:/070_Index_Mgmt/40_Custom_Dynamic_Mapping.asciidoc

* review修改

*  二次review后修改

* 组间review后修改

yichao2015 committed:
chapter10_part14:/070_Index_Mgmt/55_Aliases.asciidoc (#304)

* chapter10_part14:/070_Index_Mgmt/55_Aliases.asciidoc

* review后修改

* 二次review后提交

* 组间review后修改

xuej committed:
chapter1_part20:/010_Intro/20_Document.asciidoc (#368)

* fix

* fix

* fix

luotitan committed:
chapter24_part2: /270_Fuzzy_matching/20_Fuzziness.asciidoc (#126)

* 第一次提交

* 根据node review意见修改

richardwei2008 committed:
chapter14_part10: /110_Multi_Field_Search/50_Cross_field.asciidoc (#98)

* chapter14_part10: /110_Multi_Field_Search/50_Cross_field.asciidoc

初译

* improve

1. file name
2. “那种方案” -> “哪种方案”

* improve

* compile error fix

* improve

richardwei2008 committed:
chapter14_part7: /110_Multi_Field_Search/35_Entity_search.asciidoc (#95)

* chapter14_part7: /110_Multi_Field_Search/35_Entity_search.asciidoc

初译

* file name tag

* improve

lxy4java committed:
chapter19_part1: /210_Identifying_Words/00_intro.asciidoc  (#91)

* 粗提交

* 修改。

* review 修改

richardwei2008 committed:
chapter14_part8: /110_Multi_Field_Search/40_Field_centric.asciidoc (#96)

* chapter14_part8: /110_Multi_Field_Search/40_Field_centric.asciidoc

初译

* improve

richardwei2008 committed:
chapter17_part5: /170_Relevance/25_Query_scoring.asciidoc (#116)

* chapter17_part5: /130_Partial_Matching/25_Query_scoring.asciidoc

初译

* improve

lxy4java committed:
chapter19_part5: /210_Identifying_Words/40_ICU_tokenizer.asciidoc (#130)

* 未完成，粗提交

* 粗提交

* 格式崩了 调整下

* 崩坏调整

* review 修改。

* review修改

lxy4java committed:
chapter19_part6: /210_Identifying_Words/50_Tyding_text.asciidoc (#132)

* 未完成 粗提交

* 粗提交

* 格式整理

* 格式整理

* review修改

* review 修改

rockybean committed:
chapter42_part1:/402_Nested/30_Nested_objects.asciidoc (#173)

* chapter42_part1:/402_Nested/30_Nested_objects.asciidoc

* improve chapter42_part1:/402_Nested/30_Nested_objects.asciidoc

* improve chapter42_part1:/402_Nested/30_Nested_objects.asciidoc

pengqiuyuan committed:
chapter23_part6:/260_Synonyms/60_Multi_word_synonyms.asciidoc (#175)

* chapter23_part6:/260_Synonyms/60_Multi_word_synonyms.asciidoc

chapter23_part6:/260_Synonyms/60_Multi_word_synonyms.asciidoc

* chapter23_part6:/260_Synonyms/60_Multi_word_synonyms.asciidoc

fix

pengqiuyuan committed:
chapter23_part7:/260_Synonyms/70_Symbol_synonyms.asciidoc (#176)

chapter23_part7:/260_Synonyms/70_Symbol_synonyms.asciidoc

rockybean committed:
chapter43_part6:/404_Parent_Child/65_Grandparents.asciidoc (#288)

* chapter43_part6:/404_Parent_Child/65_Grandparents.asciidoc

* improve chapter43_part6:/404_Parent_Child/65_Grandparents.asciidoc

* improve chapter43_part6:/404_Parent_Child/65_Grandparents.asciidoc

* improve chapter43_part6:/404_Parent_Child/65_Grandparents.asciidoc

pengqiuyuan committed:
chapter22_part21:/300_Aggregations/120_breadth_vs_depth.asciidoc (#294)

* chapter22_part21:/300_Aggregations/120_breadth_vs_depth.asciidoc

初译

* chapter22_part21:/300_Aggregations/120_breadth_vs_depth.asciidoc

修改

* chapter22_part21:/300_Aggregations/120_breadth_vs_depth.asciidoc

添加第一行

* chapter22_part21:/300_Aggregations/120_breadth_vs_depth.asciidoc

按 review 意见修改。@qindongliang

* chapter22_part21:/300_Aggregations/120_breadth_vs_depth.asciidoc

按 review 意见修改。@qindongliang

* chapter22_part21:/300_Aggregations/120_breadth_vs_depth.asciidoc

按 review 意见修改。1、移除标题括号中的英文内容。2、移除 No

pengqiuyuan committed:
chapter22_part5:/300_Aggregations/30_histogram.asciidoc (#295)

* chapter22_part5:/300_Aggregations/30_histogram.asciidoc

初译

* chapter22_part5:/300_Aggregations/30_histogram.asciidoc

第一行：[[_building_bar_charts]]

* chapter22_part5:/300_Aggregations/30_histogram.asciidoc

移除多余前缀

* chapter22_part5:/300_Aggregations/30_histogram.asciidoc

按 review 意见修改。感谢 @smilesfc

* chapter22_part5:/300_Aggregations/30_histogram.asciidoc

按 review 意见修改。感谢 @qindongliang

* chapter22_part5:/300_Aggregations/30_histogram.asciidoc

按照 review 意见修改。

lephix committed:
chapter2_part5:/020_Distributed_Cluster/20_Add_failover.asciidoc (#308)

* 第二章第5部分

* 按照reviewer建议进行修改

* 按照要求增加文件名，green部分修复

lephix committed:
chapter2_part6:/020_Distributed_Cluster/25_Scale_horizontally.asciidoc (#311)

* 第二章25小节

* 按照建议进行修改

* 恢复((()))隐藏内容

lephix committed:
chapter2_part8:/020_Distributed_Cluster/35_Coping_with_failure.asciidoc (#319)

* 第二章35小节

* 消除多余的空行

* 还原去掉的((()))的隐藏内容

* 按照要求增加[[]]文件名

pengqiuyuan committed:
chapter22_part22:/300_Aggregations/125_Conclusion.asciidoc (#369)

most 翻译不准确，重提 pr！@medcl

luotitan committed:
chapter41_part4: /400_Relationships/22_Top_hits.asciidoc (#352)

* 重新提交

* 按照Medcl和node review意见修改

richardwei2008 committed:
chapter17_part11: /170_Relevance/55_Random_scoring.asciidoc (#122)

* chapter17_part11: /130_Partial_Matching/55_Random_scoring.asciidoc

初译

* improve

* improve

richardwei2008 committed:
chapter17_part3: /170_Relevance/15_Practical_scoring.asciidoc (#114)

* chapter17_part3: /130_Partial_Matching/15_Practical_scoring.asciidoc

初译

* improve

* improve

richardwei2008 committed:
chapter13_part6: /100_Full_Text_Search/25_Boosting_clauses.asciidoc (#80)

* chapter13_part6: /100_Full_Text_Search/25_Boosting_clauses.asciidoc

初译

* improve

* file name tag

* improve

规范 -> 归一

* improve

Commit:
修改stemmed翻译不当

Commit:
修改stemmed翻译不当

Commit:
删掉多余英文

Commit:
删掉多余英文

Commit:
Merge branch 'chapter/chapter6_part4' of https://github.com/sdlyjzh/elasticsearch-definitive-guide into chapter/chapter6_part4

medcl commented:
rebase一下吧

Commit:
Merge branch 'chapter/chapter6_part4' of https://github.com/sdlyjzh/elasticsearch-definitive-guide into chapter/chapter6_part4

tangmisi commented:
pr重新提交才行

sdlyjzh commented:
已重新提交 pr :405
